hooks/install

```diff
@@ -2,6 +2,12 @@
 
 set -ex
 
+RELEASE=$(lsb_release -r -s |awk '{print $1}' |awk -F. {'print $1}')
```

Comment:
Although it works, the second-to-last single quote should be before the opening curly brace.